src/Protocol/Protocol_1_8.cpp

```diff
@@ -2272,17 +2275,6 @@ bool cProtocol_1_8_0::HandlePacket(cByteBuffer & a_ByteBuffer, UInt32 a_PacketTy
 			}
 			break;
 		}
-		default:
```

Comment:
m_State is currently user-controlled from cMultiVersionProtocol, would need to add some checks

Reply:
You shouldn't use enum types to represent values that might be invalid enum values.